app/migrations/20240104144355-FlatInventory.cjs

```diff
@@ -0,0 +1,344 @@
+    /// InventoryValue ///
+    return queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.renameTable("SubCategoryValue", "InventoryValue", {
+        transaction,
+      });
+      await renameColumns(
+        "InventoryValue",
+        {
+          subcategory_value_id: "id",
+          total_emissions: "co2eq",
+          subcategory_id: "sub_category_id",
+        },
+        queryInterface,
+        transaction,
+      );
+      await queryInterface.changeColumn(
+        "InventoryValue",
+        "co2eq",
+        Sequelize.BIGINT,
+        { transaction },
+      );
+      await queryInterface.addColumn(
+        "InventoryValue",
+        "co2eq_years",
+        Sequelize.INTEGER,
+        { transaction },
+      );
+      await queryInterface.addColumn(
+        "InventoryValue",
+        "gpc_reference_number",
+        Sequelize.STRING,
+        { transaction },
+      );
+      await queryInterface.addColumn(
+        "InventoryValue",
+        "sector_id",
+        Sequelize.UUID,
+        { transaction },
+      );
+      await queryInterface.addColumn(
+        "InventoryValue",
+        "unavailable_reason",
+        Sequelize.TEXT,
+        { transaction },
+      );
+      await queryInterface.addColumn(
+        "InventoryValue",
+        "unavailable_explanation",
+        Sequelize.TEXT,
+        { transaction },
+      );
+      await queryInterface.addConstraint("InventoryValue", {
+        type: "foreign key",
+        name: "FK_InventoryValue.sector_id",
+        fields: ["sector_id"],
+        references: {
+          table: "Sector",
+          field: "sector_id",
+        },
+        onDelete: "CASCADE",
+        onUpdate: "CASCADE",
+        transaction,
+      });
+      await queryInterface.addColumn(
+        "InventoryValue",
+        "sub_sector_id",
+        Sequelize.UUID,
+        { transaction },
+      );
+      await queryInterface.addConstraint("InventoryValue", {
+        type: "foreign key",
+        name: "FK_InventoryValue.sub_sector_id",
+        fields: ["sub_sector_id"],
+        references: {
+          table: "SubSector",
+          field: "subsector_id",
+        },
+        onDelete: "CASCADE",
+        onUpdate: "CASCADE",
+        transaction,
+      });
+      await removeColumns(
+        "InventoryValue",
+        [
+          "co2_emissions_value",
+          "ch4_emissions_value",
+          "n2o_emissions_value",
+          "emissions_factor_id",
+          "emission_factor_value",
+          "subsector_value_id",
+          "sector_value_id",
+        ],
+        queryInterface,
+        transaction,
+      );
+      await queryInterface.dropTable("SubSectorValue", { transaction });
+      await queryInterface.dropTable("SectorValue", { transaction });
+
+      /// EmissionsFactor ///
+      await renameColumns(
+        "EmissionsFactor",
+        {
+          emissions_factor_id: "id",
+          emissions_factor: "emissions_per_activity",
```

Comment:
Hmm. OK. I think "emissions factor" is the term of art here. Are you just changing it to keep it from conflicting with the name of the table?

Reply:
I changed it since that's what you decided to call it in the diagram we made. I can change it back of course in case emissions_factor is better.